### PR TITLE
storage: Use semantic HTML in verify key modal

### PR DESCRIPTION
**Pull request by garrett**
Follow-up to #15844, to fix the HTML.

Would've been in #15870, but I wanted the previous fix to be implemented quickly. This changes a lot more.

### CODE REVIEW

**Comment by garrett** [comment]
It doesn't look _so much_ different, but anyone using a screen reader would care.

Screenshot from 2021-05-26 16-52-00

**Review by allisonkarlitskaya — COMMENTED** [review]
Can you add a copy widget to those while you're at it?

**Comment by garrett** [comment]
> Can you add a copy widget to those while you're at it?

I'd love that.

Specifically inline compact code: https://www.patternfly.org/v4/components/clipboard-copy#inline-compact-code

**Review by mvollmer — APPROVED** [review]

**Comment by mvollmer** [comment]
Thanks!

**Review by KKoukiou — COMMENTED** [review]
This PR also add some style fixes which are not mentioned in the commit message. Especially the 'pre' customization I would like to check before we merge this.

File: pkg/storaged/storage.scss
Comment (KKoukiou) [inline pkg/storaged/storage.scss:415]: OK Patternfly does not style ` <pre> ` with grey background etc.
Instead for code blocks or text areas they provide their own components - see https://www.patternfly.org/v4/components/code-block 

As this is overriding default PF styles I think this is not correct do here - as we 're trying to move away from custom stylesheets.

**Review by KKoukiou — CHANGES_REQUESTED** [review]

**Comment by garrett** [comment]
I've changed the pre to an inline code copy from PF @ 8895962.

However, it looks like this:

Screenshot from 2021-05-27 18-35-33

And it should be inline, with code, and have a grey background like this:

Screenshot_2021-05-27 PatternFly 4 • Clipboard copy

From https://www.patternfly.org/v4/components/clipboard-copy/#inline-compact-code

Do we need to update PF to get the new variants perhaps?

**Comment by KKoukiou** [comment]
For the inline copy clipbord component we need https://github.com/patternfly/patternfly-react/pull/5629
which is included in @patternfly/react-core@4.108.0
We currently have lower version than this so it's expected that you can't use it with this patternfly version @garrett .

**Comment by garrett** [comment]
Added a blocked tag, as we're blocked on a PF upgrade. Hopefully we can upgrade soon and get this merged in shortly after.

**Review by KKoukiou — APPROVED** [review]

**Comment by martinpitt** [comment]
Whoops, this slipped through the cracks, sorry! The PF update landed a month ago, and this was already approved. I pushed a rebase and I'll see to get this into today's release.

**Comment by martinpitt** [comment]
I tested the rebase. The dialog looks fine to me, and copy&paste works as expected.

image

**Review by martinpitt — APPROVED** [review]

**Review by mvollmer — APPROVED** [review]

**Pull request merged**
